core/src/com/mygdx/pirategame/WorldContactListener.java

```diff
@@ -105,8 +113,20 @@ public void beginContact(Contact contact) {
      */
     @Override
     public void endContact(Contact contact) {
+        Fixture fixA = contact.getFixtureA();
+        Fixture fixB = contact.getFixtureB();
+
+        int cDef = fixA.getFilterData().categoryBits | fixB.getFilterData().categoryBits;
         // Displays contact message
         Gdx.app.log("End Contact", "");
```

Comment:
Out of interest... is this event tracking?? If so - that's cool

Reply:
Yeah, I mean, it tracks collisions event if that's what you mean